ZuluContent/Zulu/Mobiles/Base/BaseElementalLord.cs

```diff
@@ -0,0 +1,29 @@
+using Server;
+using Server.Mobiles;
+
+namespace Server.Mobiles
+{
+    public class BaseElementalLord : BaseCreatureTemplate
+    {
+        [Constructible]
+        public BaseElementalLord(string templateName) : base(templateName)
+        {
+        }
+
+        public BaseElementalLord(Serial serial) : base(serial)
+        {
+        }
+        
+        public override void Serialize(IGenericWriter writer)
```

Comment:
Mmm wondering if we should use the codegen stuff for these already? I've been upgrading mine slowly as I go and when I add new items